app/controllers/report_controller.rb

```diff
@@ -31,16 +31,6 @@ def data
   def content_types
   end
 
-  def download
-    fields = params["fields"].present? ? params.delete("fields").split(/\s*,\s*/) : nil
-    params[:per_page] = 10
-    params[:page] = 1
-    response.headers["Content-Type"] = "application/octet-stream"
-    response.headers["Content-Disposition"] = "attachment; filename=report.csv"
-    response.headers["Last-Modified"] = Time.now.utc.rfc2822 # HTTP requires GMT date/time
-    self.response_body = Report.new(params, fields, current_user:).to_csv
-  end
```

Comment:
No longer needed, since the simple-datatables library will export to CSV on the client-side for us. (Would be good for @andrewjbtw to take a gander at the CSV to make sure it looks/works as expected.)